pyaerocom/io/read_eea_aqerep_base.py

```diff
@@ -627,6 +627,11 @@ def read(self, vars_to_retrieve=None,
                 self.logger.warning('{} has TemporalResolutionError'.format(_file))
                 const.print_log.warning(f'{repr(e)}. Skipping file...')
                 continue
+            except Exception as e:
```

Comment:
It might be better to just catch the UnicodeDecodeError error, but in principle I am fine with this change

Reply:
Yes, I agree, it is safer. I will change that and push a new version